lib/rest/RequestHandler.js

```diff
@@ -257,7 +257,7 @@ class RequestHandler {
                         }
 
                         if(resp.statusCode !== 429) {
-                            this._client.emit("debug", `${body && body.content} ${now} ${route} ${resp.statusCode}: ${latency}ms (${this.latencyRef.latency}ms avg) | ${this.ratelimits[route].remaining}/${this.ratelimits[route].limit} left | Reset ${this.ratelimits[route].reset} (${this.ratelimits[route].reset - now}ms left)`);
+                            this._client.emit("debug", `${body && body.content ? `${body.content} ` : ""}${now} ${route} ${resp.statusCode}: ${latency}ms (${this.latencyRef.latency}ms avg) | ${this.ratelimits[route].remaining}/${this.ratelimits[route].limit} left | Reset ${this.ratelimits[route].reset} (${this.ratelimits[route].reset - now}ms left)`);
```

Comment:
Surely you'd stringify the body object, no?

Reply:
Ah, I didn't check for that! I'll fix it in a bit.

Reply:
I may of mis-interpreted what you were trying to say now I think about it, no I haven't stringified the `body` object since I don't think it was really needed to do so and could cause some faulty type errors in the class.

Reply:
Whether it's needed is debatable, for the errors, just check `typeof body === "object"` and use it in a ternary expression